tests/tt_metal/tt_metal/unit_tests_fast_dispatch/command_queue/test_worker_config_buffer.cpp

```diff
@@ -0,0 +1,39 @@
+// SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
```

Comment:
```suggestion
// SPDX-FileCopyrightText: © 2024 Tenstorrent Inc.
```

Reply:
This SPDX license checker wasn't happy with this (I don't know exactly why).

Reply:
Hmm that's weird